OlinkAnalyze/R/pca_plot.R

```diff
@@ -376,21 +492,51 @@ olink_pca_plot <- function (df,
 
     pca_plot <- pca_plot +
       ggplot2::geom_segment(data = loadings,
-                   ggplot2::aes(x = 0,
-                       y = 0,
-                       xend = LX*loadings_scaling_factor,
-                       yend = LY*loadings_scaling_factor),
-                   arrow = ggplot2::arrow(length = grid::unit(1/2, "picas")),
-                   color = "black") +
+                            ggplot2::aes(x = 0,
+                                         y = 0,
+                                         xend = LX*loadings_scaling_factor,
+                                         yend = LY*loadings_scaling_factor),
+                            arrow = ggplot2::arrow(length = grid::unit(1/2, "picas")),
+                            color = "black") +
       ggrepel::geom_label_repel(data = loadings,
-                       ggplot2::aes(x = LX*loadings_scaling_factor,
-                           y = LY*loadings_scaling_factor,
-                           label = variables),
-                       box.padding = 1,
-                       show.legend = FALSE,
-                       segment.colour = 'gray')
+                                ggplot2::aes(x = LX*loadings_scaling_factor,
+                                             y = LY*loadings_scaling_factor,
+                                             label = variables),
+                                box.padding = 1,
+                                show.legend = FALSE,
+                                segment.colour = 'gray')
+  }
+
+  #Label outliers in figure
+  if(!is.na(outlierDefX) & !is.na(outlierDefY)){
```

Comment:
Same issue here but should be fixed if we just have a error handling section in the beginning